### DISCUSSION TITLE
error with next-auth

### DISCUSSION THREAD

**Discussion by pogran** [vercel/vercel#8679] [Help]
I have problem with next-auth. I set in my package.json 

 "vercel-build": "prisma generate && next build" and after this I don't understand why NEXTAUTH_URL doesn't work. I have 
 [next-auth][error][CLIENT_FETCH_ERROR] 

I think problem with vercel-build and need to set up env for vercel

**Comment by amyegan** [comment]
Be sure to check that you have added the appropriate environment variables for your project. You can learn more about how to add them in the guide, "[How do I add environment variables to my Vercel project?](https://vercel.com/guides/how-to-add-vercel-environment-variables)".

If you have verified your environment variables and still see the error, please provide a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example). The reproduction will help me understand the problem better and allow me to investigate the particular code and configuration combination that is triggering this error.
